src/main/resources/migration/common/V23__drop_criteria_check_add_criterion_result.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE criteria_check;
```

Comment:
Just dropped and recreated as we don't have any data we need to keep in any envs